test/lev-top-nav/Index.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import 'jest-styled-components';
+import LevTopNav from "../../src/lev-top-nav/index";
+import renderer from 'react-test-renderer';
+
+describe('component', () => {
+  let tree, root;
+  beforeEach(() => {
+    tree = renderer.create(<LevTopNav />);
+    root = tree.root;
+  });
+
+  it('renders correctly', () => expect(tree.toJSON()).toMatchSnapshot());
+
+  it('contains 3 links with page-link class', () => {
+    let links = root.findAllByProps({className: "page-link"});
+    expect(links.length).toBe(3);
+  })
```

Comment:
This is no barrier to merging but...

I'd suggest that this test has little value and should be removed.

The reason is that our tests should test that the component does what we want. - Do we care how many links there are? What if someone adds another link in the future? They won't have done anything wrong and yet this test will suddenly fail.

Reply:
If we were going to keep it, wouldn't it also be better to just look for `<a>` elements?

Reply:
Removed test